atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/AdjustableSweepBatchConfigSource.java

```diff
@@ -23,17 +23,29 @@
 import org.slf4j.LoggerFactory;
 
 import com.google.common.base.Supplier;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.schema.stream.StreamTableType;
 import com.palantir.atlasdb.util.MetricsManager;
 import com.palantir.logsafe.SafeArg;
 
 public final class AdjustableSweepBatchConfigSource {
     private static final Logger log = LoggerFactory.getLogger(BackgroundSweeperImpl.class);
 
+    // We noticed that Sweep increases GC pressure in cassandra when sweeping the VALUE table of a StreamStore.
+    // Thus, we use a reduced default config in these scenarios.
+    private static final ImmutableSweepBatchConfig STREAM_STORE_BATCH_CONFIG = ImmutableSweepBatchConfig.builder()
```

Comment:
planning to do some testing today on whether this is still important, given https://github.com/palantir/atlasdb/pull/2754